vnext/Microsoft.ReactNative/IReactDispatcher.idl

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import "IReactPropertyBag.idl";
+
+namespace Microsoft.ReactNative {
+
+  // The delegate is used to create property value on-demand.
+  [webhosthidden]
+  delegate void ReactDispatcherCallback();
+
+  [webhosthidden]
+  interface IReactDispatcher
```

Comment:
Can this replace the IMessageQueue interface (see Desktop/ABI/MessageQueue.idl)?

Reply:
I guess, yes. I was not aware about it.

---
In reply to: [423980262](https://github.com/microsoft/react-native-windows/pull/4877#discussion_r423980262) [](ancestors = 423980262)